docs/archive/bootstrap-2.md

```
@@ -0,0 +1,71 @@
+2. 시드 코드/데이터를 먼저 수정 전, 토의 후 변경사항을 보고.
+3. 코드 수정 후 ORM이 자동으로 수행되게 진행.
+
+### 상황 2) 테이블 구조를 바꿨는데 시드 코드가 옛 구조를 사용한다
+
+증상:
+
+- INSERT/적재 중 SQL 오류 발생
+
+대응:
+
+1. 시드 코드와 스키마를 ORM을 통해서 맞춰주고 PR후 Merge 진행
```

markdownlint-cli2 (0.22.0)

[warning] Files should end with a single newline character

(MD047, single-trailing-newline)